scrape/scrape_test.go

```diff
@@ -0,0 +1,33 @@
+package scrape
```

Comment:
It might be nice to pull this out into it's own package so that when using setup() or copyTestFile() it's clear where those functions are coming from.

Reply:
ack.  might do this in the future.